### PR TITLE
Run CodeFormatter for ConsoleHost

### PR DESCRIPTION
**Pull request by iSazonov**
# PR Summary

Auto fix formatting with CodeFormatter in ConsoleHost.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Comment by PoshChan** [comment]
@iSazonov, your last commit had 3 failures in `PowerShell-CI-windows`
Validate Update-Help from the Web for all PowerShell Core modules..Validate Update-Help for module 'Microsoft.PowerShell.Diagnostics' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell Core modules..Validate Update-Help for module 'Microsoft.PowerShell.Host' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```
Validate Update-Help from the Web for all PowerShell Core modules..Validate Update-Help for module 'Microsoft.PowerShell.Management' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```

**Comment by iSazonov** [comment]
@PoshChan Please retry windows

**Comment by PoshChan** [comment]
@iSazonov, successfully started retry of `PowerShell-CI-Windows`

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ namespace Microsoft.PowerShell
     internal static class ConsoleControl
     {
 #if !UNIX
-#region structs
+        #region structs
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs]: this format seems to contradict https://github.com/PowerShell/PowerShell/pull/9402/files#diff-d60cf147d07e56d9bad113594e47c7f0R275

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs [hunk 2]
```diff
@@ -1580,8 +1580,8 @@ private string ReadLineFromConsole(bool endOnTab, string initialContent, bool ca
             }
 
 #endif
-                do
-                {
+            do
+            {
 #if UNIX
                     keyInfo = Console.ReadKey(true);
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs]: shouldn't this be moved left?

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs [hunk 3]
```diff
@@ -1595,35 +1595,35 @@ private string ReadLineFromConsole(bool endOnTab, string initialContent, bool ca
...
 
-                        break;
+                        throw new PipelineStoppedException();
                     }
 
+                    break;
+                }
+
 #if UNIX
                     if (keyInfo.Key == ConsoleKey.Enter)
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs]: also indented too far, multiple cases in this file

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs (hunk 2 above)
Comment (iSazonov) [reply src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs]: Yes, but CodeFormatter follows parsing - we parse on Windows and UNIX const does not set. So CodeFormatter doesn't format Unix code at all in whole code base. I suggest ignore this in the PR.

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs (hunk 1 above)
Comment (iSazonov) [reply src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs]: I see only one strange case in PSTelemetryMethods.cs there. Here all looks ok.

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs (hunk 2 above)
Comment (SteveL-MSFT) [reply src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs]: ignore meaning revert?

Comment (iSazonov) [reply src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs]: No, we don't format UNIX code so we have nothing to revert.

Comment (SteveL-MSFT) [reply src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs]: I see.

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**